util/reggen/reg_block.py

```diff
@@ -213,15 +222,29 @@ def _validate_async(self, name: Optional[str], clk: object) -> None:
             assert isinstance(clk, ClockingItem)
             self.clocks[name] = clk
 
-    def _handle_register(self, where: str, body: object, clocks: Clocking) -> None:
+    def _handle_register(self,
+                         where: str,
+                         body: object,
+                         clocks: Clocking,
+                         is_alias: bool = False) -> None:
```

Comment:
Do we ever call this without the last argument? If not, I'd probably remove the default value. Similarly for the other functions below.

Reply:
We do - but only in very few places. I removed all the defaults and made the calls explicit everywhere.